src/components/Search/NoResults/index.tsx

```diff
@@ -13,7 +12,7 @@ interface Props {
   isSearchDrawer: boolean;
 }
 
-const NoResults: React.FC<Props> = ({ searchQuery, searchUrl = '', isSearchDrawer }) => {
+const NoResults: React.FC<Props> = ({ searchQuery }) => {
```

Comment:
since we are no longer using `isSearchDrawer` or `searchUrl`, we should remove them from the `Props` definition.

Reply:
There is an eslint rule for this, so it is easier to notice unused props, and unused variables as well, maybe I should add both to eslint rules and to `pre-commit` in `husky`, wdyt?

Reply:
Sure that sounds like a good idea

Reply:
Let me do it in a separate PR then.